renovate.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Do we want the [renovate bot?](https://github.com/renovatebot/renovate) 

Getting it to automate updating dependencies (do note that it doesn't support shards yet) might make our lives easier

Reply:
no we do not, please remove it @xbdmHQ

Reply:
> no we do not, please remove it @xbdmHQ

Ok, done.